static/app/views/onboarding/targetedOnboarding/welcome.tsx

```diff
@@ -50,26 +48,23 @@ function InnerAction({title, subText, cta, src}: TextWrapperProps) {
   );
 }
 
-type Props = {
-  organization: Organization;
-};
-
-function TargetedOnboardingWelcome({organization}: Props) {
+function TargetedOnboardingWelcome({organization, ...props}: StepProps) {
   const source = 'targeted_onboarding';
   React.useEffect(() => {
     trackAdvancedAnalyticsEvent('growth.onboarding_start_onboarding', {
-      organization,
+      organization: organization || null,
       source,
     });
   });
 
   const onComplete = () => {
     trackAdvancedAnalyticsEvent('growth.onboarding_clicked_instrument_app', {
-      organization,
+      organization: organization || null,
```

Comment:
@Neo-Zhixing wondering why we can't guarantee the organization exists here

Reply:
StepProps.organization is required right

Reply:
@Neo-Zhixing I think somewhere it was optional in the original onboarding flow which is why this happened, I guess I should just make it required everywhere and revert this change